parsec/api/protocol/events.py

```diff
@@ -1,60 +1,61 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) AGPLv3 2019 Scille SAS
 
+from enum import Enum
+
 from parsec.serde import OneOfSchema, fields, validate
 from parsec.api.protocol.base import BaseReqSchema, BaseRepSchema, CmdSerializer
 from parsec.api.protocol.realm import RealmRoleField
 from parsec.api.protocol.invite import InvitationStatusField
 
 
-EVENTS = (
-    "pinged",
-    "realm.roles_updated",
-    "realm.vlobs_updated",
-    "realm.maintenance_started",
-    "realm.maintenance_finished",
-    "message.received",
-    "invite.status_changed",
-)
+class EventsType(Enum):
```

Comment:
Shouldn't we call this just `Events` ? (especially given we have `BackendEvents`)

Reply:
`Events` seems better

Reply:
fixed